README.md

```diff
@@ -14,7 +14,7 @@
 
 Chef Development Kit (ChefDK) brings Chef and the development tools developed by the Chef Community together and acts as the consistent interface to this awesomeness. This awesomeness is composed of:
 
-* [Chef][]
+* [Chef Infra Client][]
 * [Berkshelf][]
 * [Test Kitchen][]
 * [ChefSpec][]
```

Comment:
Did you say something about getting rid of ChefSpec in the docs? Should we remove this reference also?

Reply:
ChefSpec is still there